Services.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html style="font-size: 16px;">
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="utf-8">
+    <meta name="keywords" content="">
+    <meta name="description" content="">
+    <meta name="page_type" content="np-template-header-footer-from-plugin">

```

Comment:
Code is a little hard to read, next time please include many comment and spacing!

Reply:
It is a bit hard to read a few comments will probably help with that